routes/user.go

```diff
@@ -2331,3 +2331,79 @@ func (fes *APIServer) BlockPublicKey(ww http.ResponseWriter, req *http.Request)
 		return
 	}
 }
+
+type IsFollowingPublicKeyRequest struct {
+	PublicKeyBase58Check            string
+	IsFollowingPublicKeyBase58Check string
+}
+
+type IsFolllowingPublicKeyResponse struct {
+	IsFollowing bool
+}
+
+func (fes *APIServer) IsFollowingPublicKey(ww http.ResponseWriter, req *http.Request) {
+
+	decoder := json.NewDecoder(io.LimitReader(req.Body, MaxRequestBodySizeBytes))
+	requestData := IsFollowingPublicKeyRequest{}
+	if err := decoder.Decode(&requestData); err != nil {
+		_AddBadRequestError(ww, fmt.Sprintf(
+			"IsFollowingPublicKey: Problem parsing request body: %v", err))
+		return
+	}
+
+	var userPublicKeyBytes []byte
+	var err error
+	userPublicKeyBytes, _, err = lib.Base58CheckDecode(requestData.PublicKeyBase58Check)
+	if err != nil || len(userPublicKeyBytes) != btcec.PubKeyBytesLenCompressed {
+		_AddBadRequestError(ww, fmt.Sprintf(
+			"IsFollowingPublicKey: Problem decoding user public key %s: %v",
+			requestData.PublicKeyBase58Check, err))
+		return
+	}
+
+	// Get the public key for the user to check
+	var isFollowingPublicKeyBytes []byte
+	isFollowingPublicKeyBytes, _, err = lib.Base58CheckDecode(requestData.IsFollowingPublicKeyBase58Check)
+	if err != nil || len(isFollowingPublicKeyBytes) != btcec.PubKeyBytesLenCompressed {
+		_AddBadRequestError(ww, fmt.Sprintf(
+			"IsFollowingPublicKey: Problem decoding public key to check %s: %v",
+			requestData.IsFollowingPublicKeyBase58Check, err))
+		return
+	}
+
+	utxoView, err := fes.backendServer.GetMempool().GetAugmentedUniversalView()
+	if err != nil {
+		_AddBadRequestError(ww, fmt.Sprintf("IsFollowingPublicKey Error getting view: %v", err))
+		return
+	}
+
+	var isFollowing = false
+
+	followEntries := []*lib.FollowEntry{}
+	followEntries, err = utxoView.GetFollowEntriesForPublicKey(userPublicKeyBytes, false)
+
+	for _, followEntry := range followEntries {
+		// get the profile entry for each follower pubkey
```

Comment:
we need to check somewhere in here that `followEntry.isDeleted` is false. 
```suggestion
		if followEntry.isDeleted {
			continue
		}
		// get the profile entry for each follower pubkey
```

Reply:
@lazynina I tried to add this in my local branch, but I'm getting an error when running. Since isDeleted is not exported is is not accessible from the backend codebase.

```
routes/user.go:2390:17: followEntry.isDeleted undefined (cannot refer to unexported field or method isDeleted)
```

Reply:
that's right - apologies, disregard that comment then please.